README.md

```diff
@@ -317,6 +317,8 @@ see an example in [Tarantool GitHub](https://github.com/tarantool/tarantool) rep
 * `CCACHE*` - Config variables for ccache, such as CCACHE_DISABLE
 * `PRESERVE_ENVVARS` - a comma separated list of environment variables to
   preserve.
+* `ABUILD_KEY` - a private key for signing alpine packages generated by 
+  `abuild-keygen`
```

Comment:
AFAIS, it is base64 encoded, but the readme doesn't say it.

Reply:
BTW, your one-line instruction how to create the key (from the commit message) would be useful here.

Reply:
> AFAIS, it is base64 encoded, but the readme doesn't say it.

I see now, it is not so. It is encoded before writting to a file. Got it.

Reply:
I'll add the comment about the instruction as a separate comment and will close this one.